certbot-ci/setup.py

```diff
@@ -45,4 +45,11 @@
     packages=find_packages(),
     include_package_data=True,
     install_requires=install_requires,
+
+    entry_points={
+        'console_scripts': [
+            'certbot_test=certbot_integration_tests.utils.certbot_call:main',
+            'acme_server=certbot_integration_tests.utils.acme_server:main',
```

Comment:
nit: This is messing with my muscle memory. I regularly type `venv/bin/a<tab>` but now my shell sees both `acme_server` and `activate`. What about changing this to something like `start_acme_server` or `run_acme_server`?

Reply:
OK for `run_acme_server` as run is more associated to foreground processes than start.

Reply:
Done.